src/sass/_forgot_password.scss

```
@@ -0,0 +1,4 @@
+// Forgot Password Page
+.forgot-password-button-width {
```

Avoid giving css properties in class name
```suggestion
.forgot-password-button {
```